docs/token_authorization_api.md

```diff
@@ -1,11 +1,28 @@
 # TorchServe token authorization API
 
-## Setup
-1. Download the jar files from [Maven](https://mvnrepository.com/artifact/org.pytorch/torchserve-endpoint-plugin) 
-2. Enable token authorization by adding the `--plugins-path /path/to/the/jar/files` flag at start up with the path leading to the downloaded jar files.
+Torchserve now supports token authorization by default.
```

Comment:
I suggest we specify this in multiple places: docker, kubernetes, kserve etc.
Its better to over-communicate this basic behavior change.

Reply:
Agreed. Was planning on creating a PR of documentation updates with both security related changes to inform of new behavior to TorchServe.